media/css/firefox/new/horizon/common.less

```diff
@@ -61,7 +61,7 @@
     height: 602px;
     left: -375px;
     position: absolute;
-    width: 909px;
+    width: 880px;
```

Comment:
This is a shared template being used on multiple pages. While this does fix the bug, rather than alter the width of the graphic on multiple pages it may be better just to add `pointer-events: none;` to both `.trees-left` and `.trees-right`.